packages/riipen-ui/src/components/Chip.test.jsx

```diff
@@ -0,0 +1,136 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Chip from "./Chip";
+
```

Comment:
For all the tests, I would actually be tempted to put them all inside a `describe` block sort of like you had before:

```js
describe('<Chip>', () => {
  // all tests / it blocks in here
})
```

According to the jest docs it is good to wrap objects (in our case components) in these blocks for organization https://jestjs.io/docs/en/api#describename-fn.